spec/std/file_utils_spec.cr

```diff
@@ -130,15 +130,17 @@ describe "FileUtils" do
       with_tempfile("cp_r-test", "cp_r-test-copied") do |src_path, dest_path|
         Dir.mkdir_p(dest_path)
         File.write(File.join(dest_path, "d"), "")
+        Dir.mkdir(File.join(dest_path, "cp_r-test"))
+        Dir.mkdir(File.join(dest_path, "cp_r-test", "b"))
```

Comment:
What's the reason for pre-creating these folders in the dest path?

Reply:
First, I only called `cp_r` recursive. 

After my first commit I thought about what will happen if there already exists a directory inside the target folder. Therefore, I changed the test and added this line.

And the test failed! So I noticed, that I need to build the recursion with a second function `cp_recursive`.
Afterwards I thought it might be a good idea to keep this line in the spec.

Reply:
Trouble is, now the spec doesn't test the case when dest_path is just an empty dir. Maybe we should have both...

Reply:
I will add an additional test.